x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/entsearch/EnterpriseSearch.java

```diff
@@ -53,6 +55,8 @@
 public class EnterpriseSearch extends Plugin implements ActionPlugin, SystemIndexPlugin {
     private static final Logger logger = LogManager.getLogger(EnterpriseSearch.class);
 
+    public static final String ENDPOINT = "_engine";
```

Comment:
This felt like the most reasonable place to put the shared endpoint for now, rather than create a `RestAction` class with only one String constant.